packages/oats-runtime/src/make.ts

```diff
@@ -172,6 +174,12 @@ export function makeString(
     if (typeof value !== 'string') {
       return getErrorWithValueMsg('expected a string', value);
     }
+    if (minLength != null && value.length < minLength) {
+      return getErrorWithValueMsg('expected a string with length greater than ' + minLength, value);
+    }
+    if (maxLength != null && value.length > maxLength) {
+      return getErrorWithValueMsg('expected a string with length smaller than ' + maxLength, value);
```

Comment:
ditto .length === maxLength is allowed

Reply:
is it? it wasnt clear to me from the specs if it was inclusive or not

Reply:
I would kinda read this https://cswr.github.io/JsonSchema/spec/basic_types/ as meaning that